modules/docs/arrow-docs/docs/docs/README.md

```diff
@@ -56,6 +56,7 @@ dependencies {
     compile 'io.arrow-kt:arrow-mtl:0.7.2' //optional
     compile 'io.arrow-kt:arrow-effects:0.7.2' //optional
     compile 'io.arrow-kt:arrow-effects-rx2:0.7.2' //optional
+    compile 'io.arrow-kt:arrow-effects-reactor:0.7.2' //optional
```

Comment:
Make sure to add this line to the README in the root of the repo. It isn't the same as this one, sadly.